{
    "errors.py": {
        "modified": "2005-09-29 05:56:23",
        "size": "3.55KB",
        "md5": "9471f75152114735abf203e8da14fae1",
        "sha256": "18fe762d322d8564452a71804a0c8280ebb5031ecc98d81b5014b5e7a317eebe"
    },
    "Metadata": {
        "Content-Encoding": "ISO-8859-1",
        "Content-Type": "text/plain; charset=ISO-8859-1",
        "X-Parsed-By": [
            "org.apache.tika.parser.DefaultParser",
            "org.apache.tika.parser.csv.TextAndCSVParser"
        ],
        "X-TIKA:content_handler": "ToTextContentHandler",
        "X-TIKA:embedded_depth": "0",
        "X-TIKA:parse_time_millis": "1",
        "resourceName": "b'errors.py'"
    }
}